prov/rxm/src/rxm_init.c

```diff
@@ -44,7 +44,7 @@ int rxm_info_to_core(uint32_t version, struct fi_info *hints,
 	if (FI_VERSION_GE(version, FI_VERSION(1, 5)))
 		core_info->domain_attr->mr_mode |= FI_MR_LOCAL;
 	else
-		core_info->mode |= (FI_LOCAL_MR | FI_RX_CQ_DATA);
+		core_info->mode |= (FI_LOCAL_MR | FI_RX_CQ_DATA | FI_CONTEXT);
```

Comment:
@a-ilango Please, review it. It makes possible to work with underlying providers that requires `FI_CONTEXT`.
I added this requirement in verbs provider to be able track posted operations via setting WRE (contains IBV Work Request and some additional parameters) to the 1st entry of `fi_context::internal`.

Reply:
Is there a MSG EP provider that needs FI_CONTEXT?

Reply:
The previous implementation of the fix for this bug in verbs relies on the fi_context. But after solving Arun's comment, the solution was re-implemented.
But, what about to stay it in the RxM provider?

Reply:
I'm fine with it staying.  I was just wondering if it is actually needed by anything.

Reply:
Ok. I'm going to split these changes into separate PR, because it's no more needed for verbs provider.
Thanks!